### ISSUE TITLE
Implement GRPO on Qwen2.5 7B but OOM

### ISSUE THREAD

**Issue by YangXiaojie1998** [OpenRLHF/OpenRLHF#901]
Hey, I am trying GRPO with Qwen 2.5 7B with 4 A6000 ada (48G) GPUs, but it is always OOM, does anyone can share proper hyperparameters and settings? I tried to activate LoRA but it failed in _broadcast_to_vllm() and I can not solve it. When I tried with out vllm, it just hang there for a long time. Great appreciation for any suggestions. Here are my settings (thoug changing bz cannot solve the problem):

> --ref_num_nodes 1 \
    --ref_num_gpus_per_node 2 \
    --reward_num_nodes 0 \
    --reward_num_gpus_per_node 0 \
    --actor_num_nodes 1 \
    --actor_num_gpus_per_node 2 \
    --pretrain $HDFS_HOME/model_hub/Qwen/Qwen2.5-7B-Instruct \
    --save_path $HDFS_HOME/checkpoints/$RUN_NAME \
    --micro_train_batch_size 2 \
    --train_batch_size 64 \
    --micro_rollout_batch_size 2 \
    --rollout_batch_size 256 \
    --temperature 0.6 \
    --n_samples_per_prompt 8 \
    --max_samples 10000 \
    --max_epochs 1 \
    --num_episodes 20 \
    --prompt_max_len 1024 \
    --generate_max_len 1 \
    --init_kl_coef 0.01 \
    --gamma 1.0 \
    --use_kl_estimator_k3 \
    --use_kl_loss \
    --advantage_estimator group_norm \
    --zero_stage 2\
    --bf16 \
    --actor_learning_rate 5e-7 \
    --prompt_data  $HDFS_HOME/data/data.json \
    --input_key input\
    --normalize_reward \
    --flash_attn \
    --save_steps 4 \
    --load_checkpoint \
    --wandb_run_name $RUN_NAME \
    --ckpt_path $HDFS_HOME/checkpoints/$RUN_NAME  \
    --max_ckpt_num 20000 \
    --vllm_num_engines 2 \
    --vllm_tensor_parallel_size 1 \
    --vllm_sync_backend nlcc \
    --adam_offload \
    --colocate_actor_ref \

**Comment by hijkzzz** [comment]
2 vllm 1 actor 1 ref enable adam offload
